static/app/views/replays/detail/layout/fluidGrid.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from '@emotion/styled';
+
+import {space} from 'sentry/styles/space';
+
+/**
+ * The default grid template is `max-content 1fr`, feel free to override it
+ * with a prop like: `style={{gridTemplateRows: 'auto 1fr auto'}}`
```

Comment:
Better to "extend" the styled component imo, `style` should be used for dynamic values.